# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/openjdk/armeb.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- jdk21u-jdk-21.0.3-ga/make/autoconf/platform.m4.vanilla	2024-12-06 12:27:30.179963945 +0100
+++ jdk21u-jdk-21.0.3-ga/make/autoconf/platform.m4	2024-12-06 12:28:28.526630236 +0100
@@ -54,6 +54,12 @@
       VAR_CPU_BITS=64
       VAR_CPU_ENDIAN=little
       ;;
+    arm*eb)
+      VAR_CPU=arm
+      VAR_CPU_ARCH=arm
+      VAR_CPU_BITS=32
+      VAR_CPU_ENDIAN=big
+      ;;
     arm*)
       VAR_CPU=arm
       VAR_CPU_ARCH=arm
